Sources/PT.PM.MySqlParseTreeUst/MySqlAntlrConverter.cs

```diff
@@ -0,0 +1,3043 @@
+        }
+
+        public Ust VisitCreateUdfunction([NotNull] MySqlParser.CreateUdfunctionContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitCreateUser([NotNull] MySqlParser.CreateUserContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitCreateUserMysqlV56([NotNull] MySqlParser.CreateUserMysqlV56Context context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitCreateUserMysqlV57([NotNull] MySqlParser.CreateUserMysqlV57Context context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitCreateView([NotNull] MySqlParser.CreateViewContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitCurrentSchemaPriviLevel([NotNull] MySqlParser.CurrentSchemaPriviLevelContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitCurrentTimestamp([NotNull] MySqlParser.CurrentTimestampContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitCursorStatement([NotNull] MySqlParser.CursorStatementContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitDataType([NotNull] MySqlParser.DataTypeContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitDataTypeBase([NotNull] MySqlParser.DataTypeBaseContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitDataTypeFunctionCall([NotNull] MySqlParser.DataTypeFunctionCallContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitDdlStatement([NotNull] MySqlParser.DdlStatementContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitDeallocatePrepare([NotNull] MySqlParser.DeallocatePrepareContext context)
+        {
+            var funcId = (IdToken)ExtracLiteral(context.dropFormat);
+            var arg = ConvertToInOutArgument(context.uid());
+            return new InvocationExpression(funcId, new ArgsUst(arg), context.GetTextSpan());
+        }
+
+        public Ust VisitDecimalLiteral([NotNull] MySqlParser.DecimalLiteralContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitDecimalMasterOption([NotNull] MySqlParser.DecimalMasterOptionContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitDeclareCondition([NotNull] MySqlParser.DeclareConditionContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitDeclareCursor([NotNull] MySqlParser.DeclareCursorContext context)
+        {
+            return VisitChildren(context);
```

Comment:
We don't have vulnerabilities related to cursors (specifically *Unclosed cursor*) ?